### ISSUE TITLE
Add localized Title/Description for Catalog generic purpose catalog entries (file-manager)

### ISSUE THREAD

**Issue by rgaudin** [offspot/offspot-config#31]
catalog.json needs to support i18n metadata ; at least for generic apps such as file-manager

**Comment by kelson42** [comment]
I wonder if this would not be better to handle this outside this json!?

**Comment by rgaudin** [comment]
Yes this is a draft-converted issue. the need is the localized strings. the mean is TBD.
